examples/fleet/main.go

```diff
@@ -85,18 +86,18 @@ func main() {
 		func(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 			log := log.FromContext(ctx)
 
-			cluster, err := mgr.GetCluster(ctx, req.ClusterName)
+			cl, err := mgr.GetCluster(ctx, req.ClusterName)
 			if err != nil {
 				return reconcile.Result{}, err
 			}
-			client := cluster.GetClient()
+			client := cl.GetClient()
 
 			// Retrieve the pod from the cluster.
 			pod := &corev1.Pod{}
 			if err := client.Get(ctx, req.NamespacedName, pod); err != nil {
 				return reconcile.Result{}, err
 			}
-			log.Info("Reconciling pod", "name", pod.Name, "uuid", pod.UID)
+			log.Info(fmt.Sprintf("Retrieved pod %s:>%s/%s", cl.Name(), pod.Namespace, pod.Name))
```

Comment:
this is an anti-pattern for structural logging. Just add `"ns", pod.Namespace`.

Reply:
Fixed :)